### ISSUE TITLE
Providing the same installation flow?

### ISSUE THREAD

**Issue by chambridge** [quipucords/rho#25]
Are we planning on providing the same installation flow?
https://github.com/quipucords/rho/tree/issues/3#installation

**Comment by elyezer** [comment]
I think we want to have a RPM package but I think we need to decide if we want to continue publishing it on EPEL or we want to move it somewhere else.

I see the spec file already created https://github.com/quipucords/rho/blob/rho-2.0/rho.spec but, since I don't know much about packaging, I don't know how the RPM is built and published on EPEL. Also I think we would want to automate this as a jenkins job, so maybe a follow up issue would be one to automate the packaging.

**Comment by dozdowski** [comment]
Back in the day my group at UVA maintained a few packages.  It looks like the process is largely still the same - and convenient that it already exists within EPEL.  We just need to get one or more of the current maintainers to sponsor someone in our group in as a new/current maintainer.  I'm guessing @jlalbertson would know the best contact route?

https://admin.fedoraproject.org/pkgdb/package/rpms/rho/

**Comment by jlalbertson** [comment]
Looks like @alikins or @awood would be good contacts. @alikins works for Ansible core now and @awood works for Candlepin and I know both of them so happy to put someone from our team in contact with them.

**Comment by chambridge** [comment]
I obtained **packager** role and was made a package admin for _rpms/rho_.

**Comment by chambridge** [comment]
Issue #162 was prep for creating the RPM on COPR. This issue will be used to document the updated installation flow from our upstream COPR yum repositories. Another issue ( #194  ) will be opened for moving this RPM to EPEL, but that will happen once major development is complete on this version.
